Library/ViewModels/NoShippingPostCampaignCheckoutViewModel.swift

```diff
@@ -456,7 +457,6 @@ public class NoShippingPostCampaignCheckoutViewModel: NoShippingPostCampaignChec
 
     let completeCheckoutWithCreditCardInput: Signal<GraphAPI.CompleteOnSessionCheckoutInput, Never> = Signal
       .combineLatest(self.confirmPaymentSuccessfulProperty.signal.skipNil(), checkoutId, selectedCard)
-      .takeWhen(self.confirmPaymentSuccessfulProperty.signal.skipNil())
```

Comment:
I think we should still keep this `takeWhen` - it's probably not needed since a user shouldn't be able to change the selected card again once we dismiss this vc if checkout fails, but having "change selected card" kick off checkout is not a risk I'd like to take.